libfwupdplugin/fu-smbios.h

```diff
@@ -20,12 +20,52 @@ FuSmbios	*fu_smbios_new			(void);
 #define FU_SMBIOS_STRUCTURE_TYPE_CHASSIS	0x03
 #define FU_SMBIOS_STRUCTURE_TYPE_LAST		0x04
 
+#define FU_SMBIOS_CHASSIS_KIND_OTHER		0x01
+#define FU_SMBIOS_CHASSIS_KIND_UNKNOWN		0x02
+#define FU_SMBIOS_CHASSIS_KIND_DESKTOP		0x03
+#define FU_SMBIOS_CHASSIS_KIND_LOW_PROFILE_DESKTOP 0x04
+#define FU_SMBIOS_CHASSIS_KIND_PIZZA_BOX	0x05
+#define FU_SMBIOS_CHASSIS_KIND_MINI_TOWER	0x06
+#define FU_SMBIOS_CHASSIS_KIND_TOWER		0x07
+#define FU_SMBIOS_CHASSIS_KIND_PORTABLE		0x08
+#define FU_SMBIOS_CHASSIS_KIND_LAPTOP		0x09
+#define FU_SMBIOS_CHASSIS_KIND_NOTEBOOK		0x0A
+#define FU_SMBIOS_CHASSIS_KIND_HAND_HELD	0x0B
+#define FU_SMBIOS_CHASSIS_KIND_DOCKING_STATION	0x0C
+#define FU_SMBIOS_CHASSIS_KIND_ALL_IN_ONE	0x0D
+#define FU_SMBIOS_CHASSIS_KIND_SUB_NOTEBOOK	0x0E
+#define FU_SMBIOS_CHASSIS_KIND_SPACE_SAVING	0x0F
+#define FU_SMBIOS_CHASSIS_KIND_LUNCH_BOX	0x10
+#define FU_SMBIOS_CHASSIS_KIND_MAIN_SERVER	0x11
+#define FU_SMBIOS_CHASSIS_KIND_EXPANSION	0x12
+#define FU_SMBIOS_CHASSIS_KIND_SUBCHASSIS	0x13
+#define FU_SMBIOS_CHASSIS_KIND_BUS_EXPANSION	0x14
+#define FU_SMBIOS_CHASSIS_KIND_PERIPHERAL	0x15
+#define FU_SMBIOS_CHASSIS_KIND_RAID		0x16
+#define FU_SMBIOS_CHASSIS_KIND_RACK_MOUNT	0x17
+#define FU_SMBIOS_CHASSIS_KIND_SEALED_CASE_PC	0x18
+#define FU_SMBIOS_CHASSIS_KIND_MULTI_SYSTEM	0x19
+#define FU_SMBIOS_CHASSIS_KIND_COMPACT_PCI	0x1A
+#define FU_SMBIOS_CHASSIS_KIND_ADVANCED_TCA	0x1B
+#define FU_SMBIOS_CHASSIS_KIND_BLADE		0x1C
+#define FU_SMBIOS_CHASSIS_KIND_TABLET		0x1E
+#define FU_SMBIOS_CHASSIS_KIND_CONVERTIBLE	0x1F
+#define FU_SMBIOS_CHASSIS_KIND_DETACHABLE	0x20
+#define FU_SMBIOS_CHASSIS_KIND_IOT_GATEWAY	0x21
+#define FU_SMBIOS_CHASSIS_KIND_EMBEDDED_PC	0x22
+#define FU_SMBIOS_CHASSIS_KIND_MINI_PC		0x23
+#define FU_SMBIOS_CHASSIS_KIND_STICK_PC		0x24
+
```

Comment:
This seems like it's asking for an enum rather than a list of devices.

Reply:
Fair point, fixed.